.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Claude Code Review
 
+permissions:
```

Comment:
The patch improves security by specifying granular permissions instead of using 'write-all'. It also switches from using the default GITHUB_TOKEN to a personal token, which may provide additional capabilities.

``` 
permissions:
  contents: read
  pull-requests: write
  issues: write
```